model/vm_host.rb

```diff
@@ -14,8 +14,10 @@ class VmHost < Sequel::Model
   one_to_many :storage_devices, key: :vm_host_id
   one_to_many :pci_devices, key: :vm_host_id
   one_to_many :boot_images, key: :vm_host_id
+  one_to_many :vm_host_slices, key: :vm_host_id
+  one_to_many :vm_host_cpus, key: :vm_host_id
```

Comment:
Since we will access them through the VM host model, like `vm_host.vm_host_cpus`, naming the relationships `slices` and `cpus` makes it more readable.

```suggestion
  one_to_many :slices, key: :vm_host_id, class: :VmHostSlice
  one_to_many :cpus, key: :vm_host_id, class: :VmHostCpu
```

Reply:
I kind of assumed that the convention was to name the relation the same as the model/table it points to. If it is Ok to diverge from that convention, I am ok to change it. I have not seen an example in the codebase like what you suggest. However, I agree with you that it is more readable.

Reply:
Here's several examples:
- https://github.com/ubicloud/ubicloud/blob/3f0a635210d41b3e1c83375eed31652efdb53e5a/model/postgres/postgres_resource.rb#L10
- https://github.com/ubicloud/ubicloud/blob/3f0a635210d41b3e1c83375eed31652efdb53e5a/model/minio/minio_server.rb#L11
- https://github.com/ubicloud/ubicloud/blob/3f0a635210d41b3e1c83375eed31652efdb53e5a/model/github/github_repository.rb#L9

Reply:
OK, thank you. I can make that change.